docs/how-to/use-native-token.md

```diff
@@ -0,0 +1,273 @@
+---
+comments: true
+---
+
+This document show you how you to set up and test a native token in a CDK stack. 
+
+## Summary
+
+!!! quote "From the [Unified bridge documentation](https://github.com/0xPolygonHermez/zkevm-techdocs/blob/a6d46da98ad32ace544e5dbc31d34831f9cc1bdd/knowledge-layer/architecture/PDFs/ulxly.pdf)"
+    When a token is utilized to cover gas expenses within a layer, we refer to it as the gas token for that specific layer.
+
+    Even when employing a gas token within a layer, it remains feasible to transfer L1 ETH to that layer. In such instances, the ETH is recorded within an ERC20 contract known as W-ETH, functioning as another instance of an ERC20 token.
+
+The diagram below illustrates the interchange of assets between layers, focusing on LY as a layer of interest. 
+
+It depicts several scenarios, such as bridging an ERC20 token from mainnet to another ERC20 token in LY, bridging L1 ETH to the LY gas token, or bridging a wrapped ERC20 token living on LX to LY ETH.
+
+![Unfied bridge exchanges](../img/how-to/gas-token-img/ulxly-exchanges.drawio.png)
+
+## Set up and prerequisites
+
+1. Edit the [params.yml](https://github.com/0xPolygon/kurtosis-cdk/blob/main/params.yml) file to set `zkevm_use_gas_token_contract` to `true`. This forces a couple of changes at setup time:
+
+    1. During contract deployment, an ERC20 token is deployed on L1.
+    2. The [rollup creation parameters](https://github.com/0xPolygon/kurtosis-cdk/blob/main/templates/contract-deploy/create_rollup_parameters.json) file is modified to set the `gasTokenAddress` to the address of the L1 ERC20.
+
+2. You can now run the stack:
+
+    ```sh
+    kurtosis run --enclave cdk-v1 --args-file params.yml --image-download always .
+    ```
+
+    !!! tip
+        For full set up and deploy instructions, check out the [quickstart](../quickstart/deploy-stack.md) documentation.
+
+    It takes a few minutes to compile and deploy the full set of contracts. 
+    
+    The screenshot below shows the full set of deployed services and highlights the bridge UI, L1 RPC, and L2 RPC services which we will focus on throughout this document.
+
+    ![Deployed services](../img/how-to/gas-token-img/services.png)
```

Comment:
I think the screenshot needs to be updated. We now deploy a new component called  `zkevm-bridge-proxy-001` which sits in front of the `zkevm-bridge-ui` and it should be used to access the Bridge UI (to make it work on all OS like macOS and WSL). 

<img width="933" alt="Screenshot 2024-04-22 at 16 33 09" src="https://github.com/0xPolygon/kurtosis-cdk/assets/28714795/ba92d414-bdbe-4abb-b396-6271a89ffb68">

Reply:
Can you provide an update screenshot please @leovct ?